.azure-pipelines/generation-templates/build-typewriter.yml

```diff
@@ -0,0 +1,13 @@
+steps:
```

Comment:
is this template still in use anywhere anymore?

Reply:
nice catch. removed the unused template.

Reply:
I'm using this in a stage to build typewriter which is used to generate code files against candidate clean metadata to validate whether there is an issue on the metadata or typewriter before checking in the metadata.